go/git/put.go

```diff
@@ -55,5 +57,55 @@ func PutMetadata(ctx context.Context, g *libkb.GlobalContext, arg keybase1.PutGi
 		},
 	}
 	_, err = g.GetAPI().Post(apiArg)
+
+	if err == nil {
+		err = sendChat(ctx, g, teamIDVis.TeamID, arg)
```

Comment:
Maybe go this on goroutine?

Reply:
Or use nonblock version of send message.

Reply:
(I don't care too much about this)

Reply:
Changed to use nonblock.